server-readonly/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j</artifactId>
+		<version>3.6.1-SNAPSHOT</version>
+	</parent>
+	<properties>
+		<spring.boot.version>2.4.3</spring.boot.version>
+	</properties>
+	<artifactId>rdf4j-http-server-readonly</artifactId>
```

Comment:
Just thinking that we may want a more specific name for this. What name depends a little on the direction we want to take this feature: do we want to reserve this for _just_ a simple sparql read-only endpoint, or should we consider slowly building this out to (eventually) become a full replacement for the (aging) rdf4j-server implementation?

Reply:
I think the rdf4j-server should be able to use this component as a part of it's total package. Not being the greatest front-end developer myself. I was hoping to be able to move much of the UI part to pure static JS/HTML pages using the triplyfy and other rdf.js parts.

Reply:
I'm not too hung up on the frontend side of RDF4J Server to be honest. It's mostly just a screen showing system config details and logging info, and I'm not even certain this is something we want to continue supporting as a feature of the webapp (there are some obvious security concerns with exposing that kind of info on the web).